### DISCUSSION TITLE
Notification window font is too small

### DISCUSSION THREAD

**Discussion by Asparkcn** [prasanthrangan/hyprdots#563] [Q&A] [answered]
Notification window font is too small
Where is the configuration file for this window?
![image](https://github.com/prasanthrangan/hyprdots/assets/125042452/596f7ce5-23cb-4acc-bee8-2b7dbc3fb67a)

**Comment by kRHYME7** [comment]
are you on hidpi? is your scaling looks small too? Maybe you could add a monitor specif line on your userprefs.conf ``` monitor = ,preferred,auto,2 ```

**Comment by Asparkcn** [comment]
yes, I am using HDPI, but a 2x scaling is too much. Additionally, adjusting the system scale causes some software to appear blurry, so I need to individually set the scaling for each application

**Reply by akashgagda** [reply]
Edit dunst configuration font size

**Reply by Asparkcn** [reply]
Is it this line? I tried to modify it, but it doesn't seem to work.
![image](https://github.com/prasanthrangan/hyprdots/assets/125042452/f494ff37-e482-464b-ad2e-d268e4090b75)

**Reply by akashgagda** [reply]
It's work for me

**Comment by hoodust** [comment]
For anyone wondering about this, editing dunst.conf does work, BUT you have to change themes for it to get applied. Doing so overwrites dunstrc which contains the settings that are actually used. You don't want to change it in dunstrc or else they'll revert to what's in dunst.conf next time you change themes.

**Comment by mahaveergurjar** [comment, marked as answer]
After changing the font size you need to kill and restart the Dunst 
This is also applied automatically if you change wallpapers or  themes because when the theme or wallpaper changes the dunst kill and restarts automatically by script that is located in .config/hypr/scripts/wallbashdunst.sh

you can also run this command -> killall dunst && dunst &

![231225_11h33m48s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/dea58843-c90f-47b4-850e-7e417bd932be)

![231225_11h40m23s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/2faece73-3345-45d4-8500-783d3da9c10a)
